recipes/vmd-python/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "vmd-python" %}
+{% set version = "2.0.10" %}
+{% set sha256 = "ac25c7291208868be51ef1cbb9235e70b0b96f30fcd2ec5eb3af3e31ebd66ca9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-v{{ version }}.tar.gz
+  url: https://github.com/Eigenstate/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - perl
+    - make
+    - toolchain
+    - nomkl
```

Comment:
You probably don't want to use this. If the package links to BLAS directly, pin `openblas 0.2.20` in both `build` and `run`; it doesn't seem from a glance like it does, though, so just `numpy` should be fine.